### ISSUE TITLE
Python mlagents not working

### ISSUE THREAD

**Issue by theInsomniacGameMaker** [Unity-Technologies/ml-agents#4121]
**Trying to set up the unity package and installing mlagents seems to have failed. After installing the mlagents, I tried mlagents-learn --help, and I don't get the Unity logo as I should.** 

**To Reproduce**
In command prompt type pip3 install mlagents
After installation try mlagents-learn --help
I'm not seeing the Unity Logo

**Console logs / stack traces**
2020-06-13 18:41:51.307807: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
WARNING:tensorflow:From c:\users\jain-\appdata\local\programs\python\python38\lib\site-packages\tensorflow\python\compat\v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
usage: mlagents-learn.exe [-h] [--env ENV_PATH] [--resume] [--force] [--run-id RUN_ID] [--initialize-from RUN_ID] [--seed SEED] [--inference] [--base-port BASE_PORT]
                          [--num-envs NUM_ENVS] [--debug] [--env-args ...] [--cpu] [--width WIDTH] [--height HEIGHT] [--quality-level QUALITY_LEVEL]
                          [--time-scale TIME_SCALE] [--target-frame-rate TARGET_FRAME_RATE] [--capture-frame-rate CAPTURE_FRAME_RATE] [--no-graphics]
                          [trainer_config_path]

positional arguments:
  trainer_config_path

optional arguments:
  -h, --help            show this help message and exit
  --env ENV_PATH        Path to the Unity executable to train (default: None)
  --resume              Whether to resume training from a checkpoint. Specify a --run-id to use this option. If set, the training code loads an already trained model to
                        initialize the neural network before resuming training. This option is only valid when the models exist, and have the same behavior names as the
                        current agents in your scene. (default: False)
  --force               Whether to force-overwrite this run-id's existing summary and model data. (Without this flag, attempting to train a model with a run-id that has
                        been used before will throw an error. (default: False)
  --run-id RUN_ID       The identifier for the training run. This identifier is used to name the subdirectories in which the trained model and summary statistics are
                        saved as well as the saved model itself. If you use TensorBoard to view the training statistics, always set a unique run-id for each training
                        run. (The statistics for all runs with the same id are combined as if they were produced by a the same session.) (default: ppo)
  --initialize-from RUN_ID
                        Specify a previously saved run ID from which to initialize the model from. This can be used, for instance, to fine-tune an existing model on a
                        new environment. Note that the previously saved models must have the same behavior parameters as your current environment. (default: None)
  --seed SEED           A number to use as a seed for the random number generator used by the training code (default: -1)
  --inference           Whether to run in Python inference mode (i.e. no training). Use with --resume to load a model trained with an existing run ID. (default: False)
  --base-port BASE_PORT
                        The starting port for environment communication. Each concurrent Unity environment instance will get assigned a port sequentially, starting from
                        the base-port. Each instance will use the port (base_port + worker_id), where the worker_id is sequential IDs given to each instance from 0 to
                        (num_envs - 1). Note that when training using the Editor rather than an executable, the base port will be ignored. (default: 5005)
  --num-envs NUM_ENVS   The number of concurrent Unity environment instances to collect experiences from when training (default: 1)
  --debug               Whether to enable debug-level logging for some parts of the code (default: False)
  --env-args ...        Arguments passed to the Unity executable. Be aware that the standalone build will also process these as Unity Command Line Arguments. You should
                        choose different argument names if you want to create environment-specific arguments. All arguments after this flag will be passed to the
                        executable. (default: None)
  --cpu                 Forces training using CPU only (default: False)

Engine Configuration:
  --width WIDTH         The width of the executable window of the environment(s) in pixels (ignored for editor training). (default: 84)
  --height HEIGHT       The height of the executable window of the environment(s) in pixels (ignored for editor training) (default: 84)
  --quality-level QUALITY_LEVEL
                        The quality level of the environment(s). Equivalent to calling QualitySettings.SetQualityLevel in Unity. (default: 5)
  --time-scale TIME_SCALE
                        The time scale of the Unity environment(s). Equivalent to setting Time.timeScale in Unity. (default: 20)
  --target-frame-rate TARGET_FRAME_RATE
                        The target frame rate of the Unity environment(s). Equivalent to setting Application.targetFrameRate in Unity. (default: -1)
  --capture-frame-rate CAPTURE_FRAME_RATE
                        The capture frame rate of the Unity environment(s). Equivalent to setting Time.captureFramerate in Unity. (default: 60)
  --no-graphics         Whether to run the Unity executable in no-graphics mode (i.e. without initializing the graphics driver. Use this only if your agents don't use
                        visual observations. (default: False)

**Comment by xiaomaogy** [comment]
Hi @theInsomniacGameMaker could you please fill out the full template of the bug report, and include all of the relevant information such as the OS you are on?

**Comment by chriselion** [comment]
`mlagents-learn --help` only prints the arguments, not the logo. I'm updating the documentation here: https://github.com/Unity-Technologies/ml-agents/pull/4148

**Comment by chriselion** [comment]
I updated the documentation to reflect the actual behavior. If you have further problems, please open a new issue (and fill out the template next time).

**Comment by Harsha-Musunuri** [comment]
Hi @chriselion, I could create a executable file for 3DBall example. but upon calling the executable using python API, the app opens and freezes. Even env.reset() is not helping.. What should i do, could you please explain this ?
